packages/jest-validate/src/deprecated.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * Copyright (c) 2014-present, Facebook, Inc. All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @flow
+ */
+
+'use strict';
+
+import type {ValidationOptions} from './types';
+
+const chalk = require('chalk');
+const JEST = 'Jest';
```

Comment:
we need to make sure this is coming from somewhere else or drop it entirely if we'd like to promote usage of jest-validate outside of Jest.